staging/src/k8s.io/code-generator/examples/hack/verify-codegen.sh

```diff
@@ -40,7 +40,7 @@ if [[ $ret -eq 0 ]]; then
   echo "${DIFFROOT} up to date."
 else
   echo "${DIFFROOT} is out of date. Please run hack/update-codegen.sh"
-  exit 1
+  exit $ret
```

Comment:
this one doesn't use the unified exit path because that would change the behavior of exiting before the smoke tests below

however it does preserve the exit code now